.scripts/check-alterations-sequence.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This script runs a task to check alteration files seqence:
+ * new files should come last
+ *
+ */
+
+const { execSync } = require("child_process");
+
+const alterationFilePrefix = "packages/schemas/alterations/";
+
+const allAlterations = execSync("pnpm cli db alter list", {
+  encoding: "utf-8",
+})
+  .split("\n")
+  .filter((filename) => Boolean(filename))
+  .map((filename) => filename.replace(".js", ""));
+
+const diffFiles = execSync("git diff --name-only HEAD HEAD~1", {
+  encoding: "utf-8",
+});
+const committedAlterations = diffFiles
+  .split("\n")
+  .filter((filename) => filename.startsWith(alterationFilePrefix))
+  .map((filename) =>
+    filename.replace(alterationFilePrefix, "").replace(".ts", "")
+  );
+
+for (const alteration of committedAlterations) {
+  const index = allAlterations.indexOf(alteration);
+
+  if (index < allAlterations.length - committedAlterations.length) {
+    throw new Error(
+      `Wrong alteration sequence for last commited file: ${alteration}`
```

Comment:
```ts
"Wrong alteration sequence for committed file: ${}.\n" +
  "All timestamps of committed alteration files should be greater than the biggest one in the base branch."
```

Reply:
updated

Reply:
Can we automatically update the timestamp in script name, instead of throwing an error and manually fixing it?

Reply:
Maybe we can, but that will need to make a new commit from a CI bot, less bot commits would be safer and more easy to control?

Reply:
Well, OK. I don't really have a strong preference here. Let's do manual updates, then.